examples/eap-tls/eap-tls-auth.py

```diff
@@ -80,6 +80,9 @@ def topology():
     # can ping each other
     ap1.cmd('ovs-ofctl add-flow ap1 "priority=10,actions=in_port,normal"')
```

Comment:
this hack does not work and the clients do not ping each other

Reply:
Sorry, what do you mean with doesn't work? Can you provide some technical details?